### PR TITLE
Seek category

### PR DESCRIPTION
**Pull request by MDTue**

### CODE REVIEW

File: frontend/src/Herbs/HerbsSeekCategory.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import {useEffect, useState} from "react";
```
Comment (gossie) [inline frontend/src/Herbs/HerbsSeekCategory.tsx]: I think this file can be deleted.

File: frontend/src/Herbs/test [hunk 2]
```diff
@@ -0,0 +1,22 @@
+test('that component is deleted', (done) => {
```
Comment (gossie) [inline frontend/src/Herbs/test]: This seems to be a file without suffix. jest does not recognize this as a test. Either delete the file or change the name to execute the test.

File: frontend/src/css/GridHauptseite [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<div class="container">
```
Comment (gossie) [inline frontend/src/css/GridHauptseite]: This file does not have an ending. And it is a mixture of html and css.

File: frontend/src/css/GridLogin [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.loginContainer {
```
Comment (gossie) [inline frontend/src/css/GridLogin]: This file does not have an ending. And it is a mixture of html and css.

File: frontend/src/Herbs/test (hunk 2 above)
Comment (MDTue) [reply frontend/src/Herbs/test]: is deleted

File: frontend/src/Herbs/HerbsSeekCategory.tsx (hunk 1 above)
Comment (MDTue) [reply frontend/src/Herbs/HerbsSeekCategory.tsx]: is deleted

File: frontend/src/css/GridHauptseite (hunk 3 above)
Comment (MDTue) [reply frontend/src/css/GridHauptseite]: deleted in next version

File: frontend/src/css/GridLogin (hunk 4 above)
Comment (MDTue) [reply frontend/src/css/GridLogin]: deleted in next version

**Review by gossie — APPROVED** [review]

**Pull request merged**